sound/soc/sof/loader.c

```diff
@@ -130,6 +131,42 @@ int snd_sof_fw_parse_ext_data(struct snd_sof_dev *sdev, u32 bar, u32 offset)
 }
 EXPORT_SYMBOL(snd_sof_fw_parse_ext_data);
 
+static int xman_get_cc_info(struct snd_sof_dev *sdev,
+			    struct xman_elem_header *hdr)
+{
+	int ret;
+
+	struct xman_cc_version *cc = container_of(hdr, struct xman_cc_version,
+						  hdr);
```

Comment:
struct declarations first please and remove spurious line

Reply:
done